### DISCUSSION TITLE
Which container to select for Data Product

### DISCUSSION THREAD

**Discussion by renepajta** [Azure/data-domain-batch#75] [Q&A] [answered]
Hi all ,

I am deploying Batch data product. At Synapse config I am struggling to choose which container of Enriched/Curated data lake I  should choose. Shall I create a new container per data product or point to main data lake container? 

![image](https://user-images.githubusercontent.com/6043628/127869145-3103bc5a-e167-488c-96c8-8d6ab72f6fbb.png)

**Comment by marvinbuss** [comment, marked as answer]
Thanks @renepajta. That is a good point and we probably require some more guidance here. We are recommending to create a new container per Data Product in the **Workspace Data Lake** and use the container as default storage file system for Synapse and other services.

This container is used by Synapse to store metadata as well as logs and other kind of information. The Data Product teams can still consume data from the other data lakes by assigning the respective MSIs or user IDs to the POSIX ACL layer of the **Enriched and Curated Data Lake** of the respective Data Landing Zone. 

* Where should we include this information?
* Is this already explained well enough in the docs? 
* Should we add this to the "Deploy to Azure" frontend?

**Reply by renepajta** [reply]
I think the explanation is good enough. I would then just enhance the deployment experience to point to documentation. 

It may become overwhelming for new adopters to find all necessary information pretty fast.

**Reply by marvinbuss** [reply]
Let me check how we can guide the user optimally in the Deploy to Azure experience.

**Reply by marvinbuss** [reply]
@renepajta You can track updates here: https://github.com/Azure/data-product-batch/issues/79
